dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -373,11 +420,12 @@ public void deleteCollectionAdminGroupTest() throws Exception {
 
 
     // This is currently not supported in DSpace API
```

Comment:
Please remove this comment, as you've also removed the `@Ignore`, so this seems to be supported.